app/src/main/java/de/danoeh/antennapod/adapter/SubscriptionsRecyclerAdapter.java

```diff
@@ -203,6 +208,18 @@ public void bind(NavDrawerData.DrawerItem drawerItem) {
                 count.setVisibility(View.GONE);
             }
 
+            TypedArray res = feedTitle.getContext().getTheme().obtainStyledAttributes(
+                    new int[] { android.R.attr.textColorPrimary });
+            int textColorPrimary = res.getColor(0, 0);
```

Comment:
I think typed arrays should be recycled. You can also just use `ThemeUtils.getColorFromAttr()`